nef/Preferences/Views/PickerOptionView.swift

```diff
@@ -0,0 +1,27 @@
+//  Copyright © 2019 The nef Authors.
+
+import SwiftUI
+
+struct PickerOptionView: View {
+    let title: String
+    let items: [OptionItem]
+    
+    @Binding var selection: Int
+    
+    var body: some View {
+        HStack {
+            Text("\(title):")
+                .frame(width: PreferencesView.Layout.leftPanel, alignment: .trailing)
+            
+            Picker(selection: $selection, label: Text("")) { self.texts(from: self.items) }
```

Comment:
I think there is an initializer `Picker("", $selection) {}` to avoid the empty `Text`)`

Reply:
Ouh yeaaah!